files/bootstrap.sh

```diff
@@ -262,11 +268,15 @@ sed -i s,CLUSTER_NAME,$CLUSTER_NAME,g /var/lib/kubelet/kubeconfig
 sed -i s,MASTER_ENDPOINT,$APISERVER_ENDPOINT,g /var/lib/kubelet/kubeconfig
 ### kubelet.service configuration
 
-MAC=$(curl -s http://169.254.169.254/latest/meta-data/network/interfaces/macs/ -s | head -n 1 | sed 's/\/$//')
-TEN_RANGE=$(curl -s http://169.254.169.254/latest/meta-data/network/interfaces/macs/$MAC/vpc-ipv4-cidr-blocks | grep -c '^10\..*' || true )
-DNS_CLUSTER_IP=10.100.0.10
-if [[ "$TEN_RANGE" != "0" ]] ; then
-    DNS_CLUSTER_IP=172.20.0.10;
+if [ -z ${DNS_CLUSTER_IP+x} ]; then
```

Comment:
@drewhemm Wait a minute, should the `+x` really be there or was that just for testing?

Reply:
@mogren It's something to do with how the `if` condition handles the variable if it is not defined:

https://stackoverflow.com/questions/48926946/what-does-variablex-mean-in-bash

So it's for testing in the context of testing the condition, not development/test code

Reply:
Ah... I should have checked the docs properly. I wasn't sure about the `+`. 

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

Makes sense!